hedera-node/hedera-evm/src/main/java/com/hedera/node/app/service/evm/store/contracts/precompile/codec/EvmEncodingFacade.java

```diff
@@ -67,6 +68,14 @@ public Bytes encodeGetTokenType(final int tokenType) {
                 .build();
     }
 
+    public Bytes encodeGetTokenTypeFailure(final ResponseCodeEnum status, final int tokenType) {
```

Comment:
Q: would a smaller footprint and a more general name be an overload or a replacement of the method
```suggestion
   public Bytes encodeGetTokenType(final int tokenType) {
        return encodeGetTokenType(tokenType, SUCCESS.getNumber());
    }

    public Bytes encodeGetTokenType(final int tokenType, final ResponseCodeEnum status) {
```

Same for all the others to reduce duplication and reuse code.